pkg/cluster/template/scripts/tidb.go

```diff
@@ -24,23 +24,25 @@ import (
 
 // TiDBScript represent the data to generate TiDB config
 type TiDBScript struct {
-	IP         string
-	Port       int
-	StatusPort int
-	DeployDir  string
-	LogDir     string
-	NumaNode   string
-	Endpoints  []*PDScript
+	IP            string
+	ListenAddress string
+	Port          int
+	StatusPort    int
+	DeployDir     string
+	LogDir        string
+	NumaNode      string
+	Endpoints     []*PDScript
 }
 
 // NewTiDBScript returns a TiDBScript with given arguments
-func NewTiDBScript(ip, deployDir, logDir string) *TiDBScript {
+func NewTiDBScript(ip, listenAddress, deployDir, logDir string) *TiDBScript {
```

Comment:
How about using a similar approach with `Port`, that could avoid long argument list of the function. It's not a must, I just happen to recognized that.

Reply:
great, done.